--- ../src-base/minecraft/cpw/mods/fml/common/DummyModContainer.java
+++ ../src-work/minecraft/cpw/mods/fml/common/DummyModContainer.java
@@ -15,4 +15,3 @@
 import java.io.File;
-import java.security.cert.Certificate;
 import java.util.Collections;
 import java.util.List;
@@ -155,9 +154,4 @@
     }
 
-    @Override
-    public Certificate getSigningCertificate()
-    {
-        return null;
-    }
 
     @Override
