--- ../src-base/minecraft/cpw/mods/fml/common/FMLContainer.java
+++ ../src-work/minecraft/cpw/mods/fml/common/FMLContainer.java
@@ -15,4 +15,3 @@
 import java.io.File;
-import java.security.cert.Certificate;
 import java.util.Arrays;
 import java.util.HashMap;
@@ -305,10 +304,4 @@
 
 
-    @Override
-    public Certificate getSigningCertificate()
-    {
-        Certificate[] certificates = getClass().getProtectionDomain().getCodeSource().getCertificates();
-        return certificates != null ? certificates[0] : null;
-    }
 
     @Override
