--- ../src-base/minecraft/cpw/mods/fml/common/MinecraftDummyContainer.java
+++ ../src-work/minecraft/cpw/mods/fml/common/MinecraftDummyContainer.java
@@ -14,4 +14,3 @@
 
-import java.security.cert.Certificate;
 
 import cpw.mods.fml.common.versioning.VersionParser;
@@ -39,18 +38,3 @@
     }
 
-    @Override
-    public Certificate getSigningCertificate()
-    {
-        if (FMLLaunchHandler.side() != Side.CLIENT)
-            return null;
-
-        try
-        {
-            Class<?> cbr = Class.forName("net.minecraft.client.ClientBrandRetriever", false, getClass().getClassLoader());
-            Certificate[] certificates = cbr.getProtectionDomain().getCodeSource().getCertificates();
-            return certificates != null ? certificates[0] : null;
-        }
-        catch (Exception e){} // Errors don't matter just return null.
-        return null;
-    }
 }
