--- ../src-base/minecraft/cpw/mods/fml/common/ModContainer.java
+++ ../src-work/minecraft/cpw/mods/fml/common/ModContainer.java
@@ -13,7 +13,6 @@
 package cpw.mods.fml.common;
 
 import java.io.File;
-import java.security.cert.Certificate;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
@@ -134,5 +134,4 @@
     VersionRange acceptableMinecraftVersionRange();
 
-    Certificate getSigningCertificate();
 
     public static final Map<String,String> EMPTY_PROPERTIES = ImmutableMap.of();
